kong/information_service_test.go

```diff
@@ -0,0 +1,32 @@
+package kong
```

Comment:
This repo has simple integration tests for other services.

See the `service_service_test.go` for reference where it uses `NewTestClient()` and asserts about the responses received therefrom.

Please at least check that Get() returns no error and that the version and database are not nil in the returned message.